test/fedramp/fedramp_cluster_creation_test.go

```diff
@@ -0,0 +1,148 @@
+package fedramp
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"   // I don't understand why my editor is complaining about this import.
+	. "github.com/onsi/ginkgo/v2" // dot notation is already importing GingkoV2.
+	. "github.com/onsi/gomega"
+	ocmclient "github.com/openshift/osde2e-common/pkg/clients/ocm"
+	awscloud "github.com/openshift/osde2e-common/pkg/clouds/aws"
+	"github.com/openshift/osde2e-common/pkg/openshift/rosa"
+)
+
+var _ = Describe("Fedramp Rosa Cluster Creation", ginkgo.Ordered, Label("Fedramp"), func() {
+	var (
+		logger         = ginkgo.GinkgoLogr
```

Comment:
to fix the imports, you should be able to change these to:
```suggestion
var _ = Describe("Fedramp Rosa Cluster Creation", Ordered, Label("Fedramp"), func() {
	var (
		logger         = GinkgoLogr
```